Firestore/Source/Core/FSTSyncEngine.mm

```diff
@@ -381,6 +389,8 @@ - (void)applyRemoteEvent:(const RemoteEvent &)remoteEvent {
 }
 
 - (void)applyChangedOnlineState:(OnlineState)onlineState {
+  [self assertCallbackExistsForSelector:_cmd];
```

Comment:
Thanks for noticing this! This check is missing [from Android](https://github.com/firebase/firebase-android-sdk/blob/master/firebase-firestore/src/main/java/com/google/firebase/firestore/core/SyncEngine.java#L339), which is probably wrong. Can you mention this inconsistency in chat and potentially fix it on Android/web client if applicable?

Reply:
Done. Will fix there too.